winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with SingleCharacter](avatarview_images/winui_initials_avatarview.png)
+
+**DoubleCharacter**
+
+The `DoubleCharacter` is used for displaying a two-characters text you have set in the `AvatarName` property. If the initials consist of a single word, it will show the first and last letters of that word. If the initials contain two or more words, it will display the first letter of the first word and the first letter of the second word.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_doubleinitials_avatarview.png)
+
+## Custom Image
+
+You can add a custom image by setting the `ImageSource` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="CustomImage"
+                         AvatarSize="ExtraLarge"
+                         ImageSource="Images\person.png">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.CustomImage;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.ImageSource = new BitmapImage(new Uri("ms-appx:///Images\\person.png"));
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom image.](avatarview_images/winui_imagesource_avatarview.png)
+
+## Avatar Character
+
+You can set the pre-defined images in avatar view by setting the `AvatarCharacter` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="AvatarCharacter"
+                         AvatarSize="ExtraLarge"
+                         AvatarCharacter="Avatar15">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+            
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar15;
+            
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with avatar character](avatarview_images/winui_avatarcharacter_avatarview.png)
+
+## Group View 
+
+You can add upto three images or initials in the same view using a GroupView type.
+
```

Comment:
@MadhumithanJayapathi The `Group` content type allows you to display up to three images or initials within a single avatar view. To use this content type, you must set the following properties, which include:

Reply:
@Manivannan-E Changed the content of the group as mentioned.